specs/DesignDiscussions/DotNetTemplate.md

```diff
@@ -0,0 +1,338 @@
+# Overview
+## Templating
+![](img1.png)
+
+Template allows users to define where data binding occurs.
+
+```json
+{
+    "type": "TextBlock",
+    "text": "{latestPrice} ",
+    "size": "ExtraLarge"
+}
+```
+
+Once binding is defined, adaptive cards will be updated by changing only the data.
+```json
+{
+  "latestPrice": 128.9,
+}
+```
+### Syntax
+- Binding syntax starts with { and ends with }. E.g., {myProperty}
+- Dot-notation to access sub-objects
```

Comment:
>ot-notation to access sub-objects [](start = 3, length = 33)

Matt: Add here that we need to gracefully handle null objects (if manager is null, it should just return empty)

Reply:
it's noted. I'll also add unit tests to ensure it's handled properly